### DISCUSSION TITLE
Docs-as-code/docToolChain application on an industrial setting

### DISCUSSION THREAD

**Discussion by hcadavid** [docToolchain/docToolchain#577] [Q&A]
Hello everyone!

In the last couple of days, I came across the 'docs-as-code' concept and later discovered this project as a really interesting example of this documentation management approach. I'm interested in exploring this topic as it seems to suit some documentation issues on a multi-disciplinary domain I'm currently investigating, but I've found little information about its application in concrete scenarios/companies. So I'm wondering, have you used the docs-as-code approach and tools like this one in an actual industrial setting? If so, could you point me to information -if it is publicly available- where this can be evidenced? I believe this topic deserves more attention from the software engineering research community, so any information would help me to contribute to this purpose. Thank you in advance!

**Comment by rdmueller** [comment]
thanx four your interest in the docs-as-code approach.
The approach itself is used by many companies, for instance Facebook and Microsoft. Many have their own toolchain like https://docusaurus.io/ build by facebook. Others just use plain AsciiDoc or Markdown which is often enough.

As a solution architect, I needed some more features than just plain text and that is why I created docToolchain in the first place.
It is now used by more companies than I had ever dreamed of. DB Systel (German Railway) and TomTom are named in the readme, but I know of many other large companies who use it - but most of this information is not public available :-|
